src/dotty/tools/dotc/printing/Highlighting.scala

```diff
@@ -0,0 +1,81 @@
+package dotty.tools
+package dotc
+package printing
+
+import scala.collection.mutable
+import core.Contexts.Context
+
+object Highlighting {
+
+  implicit def highlightShow(h: Highlight)(implicit ctx: Context): String =
+    h.show
+  implicit def highlightToString(h: Highlight): String =
```

Comment:
Not sure if I still need them...yeah, maybe I can get rid of them now...gonna check!